### DISCUSSION TITLE
Render beamer presentation without "incremental" slides

### DISCUSSION THREAD

**Discussion by maxdrohde** [quarto-dev/quarto-cli#5473] [Q&A] [answered]
In my beamer presentations, I'll use `. . .` to advance the slides incrementally. Such as

````
## First slide

This is my first point

. . .

This is my second point
````

which creates two slides. However, is it possible to set a global option that renders all the slides without this incremental feature? For example,  I want one version with incremental slides when I'm presenting, but not when I'm printing them.

I suppose I could delete all the `. . .`, but I feel like there may be a better way?

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/presentations/beamer.html#incremental-lists>.

You might also be interested in <https://quarto.org/docs/authoring/conditional.html> and <https://quarto.org/docs/projects/profiles.html>.

**Comment by cderv** [comment]
> I suppose I could delete all the . . ., but I feel like there may be a better way?

Using a Lua filter would allow you to quickly remove all the `. . .` from the document if you want a version without it.

**Comment by cl-roberts** [comment, marked as answer]
Hi there,

I wrote a simple [Lua filter](https://github.com/clroberts-adfg/lua-filters/tree/main/incremental-to-static-slides) to solve this, as suggested by @cderv. Note I've only tested it with beamer, but should also work with revealjs

**Reply by maxdrohde** [reply]
This is great, thanks for making this!
